app/presenters/vendor_api/application_presenter/hesa_data.rb

```diff
@@ -0,0 +1,23 @@
+module VendorAPI::ApplicationPresenter::HesaData
+  def schema
+    super.deep_merge!({
+      attributes: {
+        hesa_data: hesa_data,
+      },
+    })
+  end
+
+  def hesa_data
+    return nil unless ApplicationStateChange::ACCEPTED_STATES.include?(application_choice.status.to_sym)
+
```

Comment:
Should we also return `nil` if the application was from last cycle, as not all of the data from last cycle cleanly maps into the new codes?